plenum/server/consensus/view_change_service.py

```diff
@@ -150,6 +153,12 @@ def _build_view_change_msg(self):
 
     def process_view_change_message(self, msg: ViewChange, frm: str):
         result = self._validate(msg, frm)
+        if result == STASH_WAITING_VIEW_CHANGE:
```

Comment:
It would be great to have a unit test in `test_view_change_service` to make sure that `StartViewChange` is sent on a quorum of `ViewChange` messages for future view (and not sent when there is no quorum or ViewChange messages are for old or current view).
This kind of tests can help in understanding and formalizing our state machine.